### ISSUE TITLE
Mail Intake | Wrong BGS error message

### ISSUE THREAD

**Issue by kierachell** [department-of-veterans-affairs/caseflow#3604]
### Just because 
```
<?xml version='1.0' encoding='UTF-8'?><S:Envelope xmlns:S="http://schemas.xmlsoap.org/soap/envelope/"><S:Body><S:Fault xmlns:ns4="http://www.w3.org/2003/05/soap-envelope"><faultcode>S:Server</faultcode><faultstring>findFlashes:
Invalid Input</faultstring><detail><ns2:ShareException xmlns:ns2="http://services.share.benefits.vba.va.gov/"><message>findFlashes: FileNumber is empty</message></ns2:ShareException></detail></S:Fault></S:Body></S:Envelope>
```

and response:
```
Completed 422 Unprocessable Entity in 2594ms (Views: 0.5ms | ActiveRecord: 1.8ms)
```

Results in:
<img width="814" alt="screen shot 2017-10-13 at 10 59 34 am" src="https://user-images.githubusercontent.com/18075411/31552475-9fb9b214-b005-11e7-9966-8533e903655d.png">

**Comment by shanear** [comment]
@kierachell The veteran exists in this case right?

**Comment by kierachell** [comment]
- `findFlashes` comes back: `findFlashes: FileNumber is empty`, but:
`<NUMBER_OF_RECORDS>1</NUMBER_OF_RECORDS>`
- veteran exists in BGS just doesn't have all the info
- it doesn't matter - why does alert say anything about `permission`???

**Comment by shanear** [comment]
Because we use find flashes as a proxy for knowing if a user has permission to see/manipulate a veteran's data. Could you test with a sensitive veteran?

**Comment by kierachell** [comment]
- Sensitivity works fine, yes:
```
<?xml version='1.0' encoding
='UTF-8'?><S:Envelope xmlns:S="http://schemas.xmlsoap.org/soap/envelope/"><S:Header><work:WorkContext xmlns:work="http://oracle.com/weblogic/soap/workarea/">rO0ABXdUAB13ZWJsb2dpYy5hcHAuQ29ycG9yYXRlRGF0YUVBUgAAANYAAAAjd2VibG9naWMud29ya2FyZWEuU3RyaW5nV29ya0NvbnRleHQABjEwLjIuMAAA</work:WorkContext></S:Header><S:Body>
<S:Fault xmlns:ns4="http://www.w3.org/2003/05/soap-envelope"><faultcode>S:Server</faultcode><
faultstring>This is an employee record at your station</faultstring>
<detail><ns2:ShareException xmlns:ns2="http://services.share.benefits.vba.va.gov/"><message>This is an employee record at your station</message></ns2:ShareException></detail></S:Fault></S:Body></S:Envelope>
```
- along with some random ass error:
`[2017-10-16 19:59:06 -0400] Completed 422 Unprocessable Entity in 1781ms (Views: 9.2ms | ActiveRecord: 1.9ms)`

### Why does this error have the same result on the frontend:
```
<?xml version='1.0' encoding='UTF-8'?><S:Envelope xmlns:S="http://schemas.xmlsoap.org/soap/envelope/"><S:Header><work:WorkContext xmlns:work="http://oracle.com/weblogic/soap/workarea/">rO0ABXdUAB13ZWJsb2dpYy5hcHAuQ29ycG9yYXRlRGF0YUVBUgAAANYAAAAjd2VibG9naWMud29ya2FyZWEuU3RyaW5nV29ya0NvbnRleHQABjEwLjIuMAAA</work:WorkContext>
</S:Header>
<S:Body>
<S:Fault xmlns:ns4="http://www.w3.org/2003/05/soap-envelope">
<faultcode>S:Server</faultcode><faultstring>findFlashes: Invalid Input</faultstring><detail><ns2:ShareException xmlns:ns2="http://services.share.benefits.vba.va.gov/"><message>findFlashes: FileNumber is empty</message>
</ns2:ShareException></detail></S:Fault></S:Body></S:Envelope>
```

**Comment by shanear** [comment]
fixed by #3661

**Comment by kierachell** [comment]
**PASSED**
<img width="768" alt="screen shot 2017-10-19 at 7 44 59 am" src="https://user-images.githubusercontent.com/18075411/31769404-706b32d0-b4a1-11e7-9621-2a26d42cd59f.png">
